BuildingAndTestingInBunsen.md

```diff
@@ -0,0 +1,160 @@
+# Building and Testing the VDO software in a Bunsen Environment
+
+This guide is intended to help get you started with developing, building, and testing your contributions to VDO in a Bunsen environment.
+
+# Prerequisites
+
+It is expected that you have satisfied the following criteria before getting started:
+
+* [Set up a Bunsen Environment](Getting_Started.md)
+
+# Expectations of this Guide
+
+This guide is currently intended to provide enough information to get minimally functional.  A more detailed guide will be generated later, but in the meantime this document should help you get started if you're interested in contributing to the VDO projects.
+
+# Find the appropriate code repository
+
+Identify which repository you want to test.
+
+Currently the following VDO repositories are set up to allow for local testing:
+* [vdo-devel](https://github.com/dm-vdo/vdo-devel) - This is where the VDO product code resides as well as any supporting test code that is only needed for VDO itself.
+* [common](https://github.com/dm-vdo/common) - This is where the library code resides that supports both the test environment as well as VDO.
+
+* How do I tell what's the difference between the two?
+  * The first thing to figure out is whether you're modifying the VDO code itself or the tests that directly test VDO.
+    * If that's the case, then you want to use the [vdo-devel](https://github.com/dm-vdo/vdo-devel) tree.
+  * Otherwise, you'll need to look for the code in either repository.  If you're not sure, then you should likely start on the [vdo-devel](https://github.com/dm-vdo/vdo-devel) repository and move to the [common](https://github.com/dm-vdo/common) repository if you still couldn't find what you were looking for.
+
+# Find the code
+
+For the purposes of this guide, all of the work you will be doing is going to take place on the Bunsen `resource` system.
```

Comment:
Line 30: For the purposes of this guide, all work will take place on the Bunsen 'resource' system.

Reply:
Adapted and available in the latest revision of this PR.